.github/workflows/pull-request.yml

```
@@ -135,11 +135,19 @@ jobs:
   success:
     name: Success
     runs-on: ubuntu-latest
+    if: ${{ always() }}
     needs: 
       - checks
       - component
       - awx-e2e
       - eda-e2e
       # - hub-e2e
     steps:
-      - run: echo Success
\ No newline at end of file
+      - if: ${{ success() }}
+        run: |
+          echo Success
+          exit 0
+      - if: ${{ !success() }}
```

Should this be 
```suggestion
       if: ${{ failure() }}
```


https://docs.github.com/en/actions/learn-github-actions/expressions